rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -914,7 +914,7 @@ def send_arrow_chunk(
     timelines: `Dict[str, arrow::Int64Array]`
         A dictionary mapping timeline names to their values.
     components: `Dict[str, arrow::ListArray]`
-        A dictionary mapping component names to their values.
+        A dictionary mapping component types to their values.
```

Comment:
digged into this method and looks like this isn't actually true. It ends up creating descriptors https://github.com/rerun-io/rerun/blob/2f7d27ab9900494ea7d780efd967569038feccde/rerun_py/src/arrow.rs#L137 from fully formed python objects https://github.com/rerun-io/rerun/blob/2f7d27ab9900494ea7d780efd967569038feccde/rerun_py/src/arrow.rs#L27
which implies that the type annotation is also wrong? This is used as a utility for `AnyValues`

so in our